microbench/src/main/java/io/netty/microbenchmark/common/AsciiStringBenchmark.java

```diff
@@ -40,13 +43,22 @@ public class AsciiStringBenchmark extends AbstractMicrobenchmark {
     private AsciiString asciiString;
     private String string;
     private static final Random random = new Random();
+    private AsciiString connection;
+    private AsciiString Connection;
 
     @Setup(Level.Trial)
     public void setup() {
         byte[] bytes = new byte[size];
         random.nextBytes(bytes);
         asciiString = new AsciiString(bytes, false);
         string = new String(bytes, CharsetUtil.US_ASCII);
+        connection = AsciiString.cached("connection");
+        Connection = AsciiString.cached("Connection");
+    }
+
+    @Benchmark
+    public boolean equalsIgnoreCaseBench() {
+        return Connection.contentEqualsIgnoreCase(connection);
     }
```

Comment:
Does it make sense to add a string with an offset? In my minds eye not much has changed there but figure I'd bring it up.

Reply:
It depends by how things are used in the most common scenarios. I'm trying to be nice(r) which HTTP usage so I have made explicit the case I care the most, which is basically an HTTP 1.1 headers map hit where the decoded & stored header name start with an upper case letter, while the one we search for is lowercase, with both cached AsciiString(s) (offset is 0 for both).
To be completely fair, I could create a new benchmark which makes more interesting but still common http 1.1 use cases, or just leveraging the header map ones (if any), adding a case like this.